setup.py

```diff
@@ -1,3 +1,14 @@
+import sys
+if sys.version_info < (3, 6) or sys.version_info >= (3, 9):
```

Comment:
Is PEP-8 good with this? I understand the intent is to fail ASAP, so maybe you already considered this?

Reply:
I made the call that failing asap was more important than the style guide. For what it is worth, here is what pylint complains about for the code I added:
```
setup.py:1:0: C0114: Missing module docstring (missing-module-docstring)
setup.py:4:4: C0103: Constant name "py_version" doesn't conform to UPPER_CASE naming style (invalid-name)
setup.py:5:4: C0103: Constant name "error" doesn't conform to UPPER_CASE naming style (invalid-name)

```

Reply:
You can use `# noinspection PyPep8Naming` to remove those second two complaints in PyCharm if you want.

Reply:
I think it's a reasonable call @KjellSwedin